modules/Emails/Email.php

```diff
@@ -1195,8 +1195,8 @@ public function email2Send($request)
                         strlen($file)
                     ); // strip GUID	for PHPMailer class to name outbound file
 
-                    $mail->AddAttachment($fileLocation, $filename, 'base64', $this->email2GetMime($fileLocation));
-                    //$mail->AddAttachment($fileLocation, $filename, 'base64');
+                    $mail->addAttachment(stream_resolve_include_path($fileLocation), $filename, 'base64', $this->email2GetMime(stream_resolve_include_path($fileLocation)));
```

Comment:
`stream_resolve_include_path($fileLocation)` can be taken into a variable, just to avoid twice call for the same $fileLocation
`$fileLocation = stream_resolve_include_path($fileLocation)`